sqlglot/parser.py

```diff
@@ -5361,6 +5361,16 @@ def _parse_function_call(
             alias = not known_function or upper in self.FUNCTIONS_WITH_ALIASED_ARGS
             args = self._parse_csv(lambda: self._parse_lambda(alias=alias))
 
+            post_func_comments = self._curr and self._curr.comments
+            if known_function and post_func_comments:
+                # If the meta comment "/* sqlglot.meta anonymous */" is following the function
+                # call we'll construct it as exp.Anonymous, even if it's "known"
+                for comment in post_func_comments:
+                    lower = comment.lower()
+                    if exp.SQLGLOT_META in lower and "anonymous" in lower:
```

Comment:
it can be case sensitive.

Reply:
Re: case sensitivity, you mean reverting the `lower()` call? I went with it for better UX, fine either way.

To leverage `add_comments` we'd have to create a dummy node for it and attach the comments. Though, they could be valid comments for all we know so we might have to undo that. In the end, I think we still need this lookahead/peek afaict.

Reply:
The comment comes after the function so it'd be added at the end of `_parse_function_call`, i.e after validation has ran